css-stylesheets/supported-css.md

```diff
@@ -1208,6 +1208,14 @@ Unless otherwise stated, the following values are supported:
   As per CSS3 specification
   </td>
 </tr>
+<tr>
+  <td>page-break-before, page-break-after</td>
```

Comment:
Can you add a `(since 8.2.4)` notice to prevent confusion? thanks.

Reply:
> Can you add a `(since 8.2.4)` notice to prevent confusion? thanks.

done